### ISSUE TITLE
centernet face 训练时 loss = inf

### ISSUE THREAD

**Issue by ChHanXiao** [yuqj1990/caffe_train#6]
练不起来loss 一直是inf，有什么调参技巧吗
I0419 17:36:08.272820  7887 CenternetLossLayer.cpp:213] total loss: nan, loc loss: inf, conf loss: nan, normalizer: 49, num_classes: 1, output_width: 160, output_height: 160
 cuda diff_sumarize_a: nan, count: 0.000000  
 cuda diff_sumarize_a: nan, count: 0.000000  
 cuda diff_sumarize_a: nan, count: 0.000000  
 cuda diff_sumarize_a: nan, count: 0.000000  
 cuda diff_sumarize_a: nan, count: 0.000000

**Comment by yuqj1990** [comment]
是训练的人脸吗？人脸的话，我用的是widerface数据集，不应该的啊

**Comment by ChHanXiao** [comment]
也是用的widerface，SSD可以训练，就这个不行

**Comment by yuqj1990** [comment]
我测试了，是我的问题，我已将gpu版本的计算loss屏蔽了，我一开始是使用cpu版loss的，后来加上gpu版loss没有测试

**Comment by yuqj1990** [comment]
你可重新更新再试试

**Comment by yuqj1990** [comment]
另外，你在训练完测试的时候，你可能会发现和文章中的结果mAP不一致比他低，是因为论文中没有进一步做nms，我在代码中进一步为了保证结果的正确性进一步过滤。不过这种所谓的那么高的mAP没啥实际意义，一旦实际商用，以很低的置信度换得的结果根本不能用
